tests/test_resources.py

```diff
@@ -147,3 +147,16 @@ def test_force_compact(tako_env):
     t1 = make_program_task(ProgramDefinition(["hostname"]), resources=rq1)
     session.submit([t1])
     session.wait_all([t1])
+
+
+def test_hw_monitoring(tako_env):
```

Comment:
@spirali What do you recommend we add here?

Reply:
I do not understand the question. But test_resources.py has a different purpose. Please create a new one: "test_monitoring.py"